usng4j-impl/src/test/java/org/codice/usng4j/impl/CoordinateSystemTranslatorTest.java

```diff
@@ -1232,6 +1245,23 @@ public void testUsngToLatLon() throws ParseException {
     }
   }
 
+  @Test
+  public void testParsingInvalidLatitutedZonesToUsng() throws ParseException {
+    final Set<String> invalidUsngStrings =
+        new HashSet<>(
+            Arrays.asList(
+                "15A", "15B", "15I", "15O", "15Y", "15Z", "15K IC", "15K OC", "15K AI", "15K AO",
+                "15K AW", "15K AX", "15K AY", "15K AZ"));
+    for (final String invalidValue : invalidUsngStrings) {
+      try {
+        UsngCoordinateImpl.parseUsngString("18I");
+      } catch (ParseException e) {
+        continue;
+      }
+      fail(invalidValue + " is not valid USNG");
```

Comment:
:pencil2: Declaring that a string value is not valid USNG doesn't fully describe the failure; it should be mentioned that this invalid USNG string seemed to be parsed correctly.

Reply:
:pencil2: I would like to have the `UsngCoordinateImpl` that was parsed from the invalid string included in the error message. This would give testers debugging this test in case of failure a good clue as to how the invalid string was incorrectly parsed into USNG.